### PR TITLE
added basic deployment and contribution related doc

### PR DESCRIPTION
**Pull request by SrinivasChilveri**
Basic deployment guide and contribution documents are added

### CODE REVIEW

File: content/en/docs/getting-started-minikube.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
++++
```
Comment (asifdxtreme) [inline content/en/docs/getting-started-minikube.md]: Do we really need this guide over here? I guess we should not keep third party installation steps in our repo, we should just provide a link to other official website for third party installation

File: resources/_gen/assets/css/css/academic-bundle-pre.css_d3f53f09220d597dac26fe7840c31fc9.content [hunk 2]
```diff
@@ -1 +1 @@
-.featured{font-size:.8rem;line-height:1.5;color:#555;text-align:center}.featured h3{margin-top:10px;margin-bottom:10px;font-weight:400;color:#333;text-align:center}.dark .featured,.dark .featured h3{color:#fff}.featured-icon{display:block;width:100%;height:40%;font-size:3rem;text-align:center}.featured-icon img{margin:0 auto}.featured .col-sm-4 p{text-align:justify}
```
Comment (asifdxtreme) [inline resources/_gen/assets/css/css/academic-bundle-pre.css_d3f53f09220d597dac26fe7840c31fc9.content]: the resources files needs to be skipped

File: resources/_gen/assets/css/css/academic.css_187be290e8222f6bb644052568a2fb6d.content [hunk 3]
```diff
@@ -1,2 +1,2 @@
-/* Source Themes Academic v3.3.0 | https://sourcethemes.com/academic/ */
```
Comment (asifdxtreme) [inline resources/_gen/assets/css/css/academic.css_187be290e8222f6bb644052568a2fb6d.content]: ditto

**Comment by asifdxtreme** [comment]
Probably you can remove the Simple Deployment docs and just keep Contribution guide

File: content/en/docs/contribute.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+# Getting started
+
+- Read the [setup](https://github.com/volcano-sh/website/blob/master/content/en/docs/basicdeployment.md) for build/deploy instructions.
+
+
+# Your First Contribution
+
+We will help you to contribute in different areas like filing issues, developing features, fixing critical bugs and getting your work reviewed and merged.
+
+If you have questions about the development process, feel free to jump into our [Slack Channel](https://github.com/volcano-sh/volcano/blob/master/slack-invitation) or join our [mailing list](https://groups.google.com/forum/#!forum/volcano-sh).
```
Comment (asifdxtreme) [inline content/en/docs/contribute.md]: Slack channel link 404

File: resources/_gen/assets/css/css/academic-bundle-pre.css_d3f53f09220d597dac26fe7840c31fc9.content (hunk 2 above)
Comment (SrinivasChilveri) [reply resources/_gen/assets/css/css/academic-bundle-pre.css_d3f53f09220d597dac26fe7840c31fc9.content]: just need to be skipped ? or we can remove from the repo?

> the resources files needs to be skipped

just need to be skipped ? or better we can remove from the repo?

File: content/en/docs/getting-started-minikube.md (hunk 1 above)
Comment (SrinivasChilveri) [reply content/en/docs/getting-started-minikube.md]: deleted

File: resources/_gen/assets/css/css/academic-bundle-pre.css_d3f53f09220d597dac26fe7840c31fc9.content (hunk 2 above)
Comment (SrinivasChilveri) [reply resources/_gen/assets/css/css/academic-bundle-pre.css_d3f53f09220d597dac26fe7840c31fc9.content]: as discussed deleted all under _gen/css & _gen/js folders.

File: resources/_gen/assets/css/css/academic.css_187be290e8222f6bb644052568a2fb6d.content (hunk 3 above)
Comment (SrinivasChilveri) [reply resources/_gen/assets/css/css/academic.css_187be290e8222f6bb644052568a2fb6d.content]: done

File: content/en/docs/contribute.md (hunk 4 above)
Comment (SrinivasChilveri) [reply content/en/docs/contribute.md]: changed

**Pull request merged**